lib/pure/asyncdispatch.nim

```diff
@@ -1443,7 +1444,7 @@ proc recvLine*(socket: AsyncFD): Future[string] {.async, deprecated.} =
       return
     add(result, c)
 
-proc callSoon*(cbproc: proc ()) =
+asyncfutures.callSoon = proc (cbproc: proc ()) =
```

Comment:
I think we need something similar to ``getGlobalDispatcher`` for this, otherwise this will be uninitialised in other threads.